### DISCUSSION TITLE
Academic Twitter API Shutdown？

### DISCUSSION THREAD

**Discussion by Jacobzwj** [cjbarrie/academictwitteR#379] [Q&A]
Although Twitter announced its new API pricing plan in February 2023, which charges $100 per month for 10k requests and $5000 per month for 1m requests, the academic Twitter API was still functioning normally until today. 

However, this morning (2023-06-25, +08:00), I discovered that my academic Twitter API could no longer be accessed. If we attempt to use get_all_tweets() or count_all_tweets(), we receive an error message stating "something went wrong. Status code: 403". Upon checking my Twitter Developer Portal (Dashboard), I discovered that my previous academic API project had been closed. Is anyone else experiencing the same issue as me?

**Comment by SilviaDalben** [comment]
Just figured out the same happened to my account. :(
I send a request using the Support form on Academic API.
Did you solve your issue or get any further information about this?

**Comment by shmuhammadd** [comment]
I am having the same issue—unfortunate development for the research community. 

From community : https://twittercommunity.com/t/get-2-users-id-following-is-now-broken/197526
